include/dd/Package.hpp

```diff
@@ -547,6 +573,24 @@ namespace dd {
             p->setIdentity(true);
         }
 
+        vEdge makeStateFromVector(const CVec::const_iterator& begin,
+                                  const CVec::const_iterator& end,
+                                  const std::size_t           level) {
```

Comment:
Would `level` be more appropriately typed as `Qubit`?

Reply:
Agreed. Done in https://github.com/cda-tum/dd_package/pull/135/commits/d59548b5625470658fe1d982b6aaa4bfd981148e